src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs

```diff
@@ -95,6 +97,43 @@ public class InvitationsController : ControllerBase
             return this.FromResult(result);
         }
 
+        [Authorize(Roles = Permissions.IPO_SIGN)]
+        [Authorize(Roles = Permissions.IPO_WRITE)]
```

Comment:
We need to implement functionality for several roles as has been done in preservation. See for example TagsController ln 77 in preservation, the AuthorizeAny method. We can talk about it if unclear :)

Reply:
Talked to JI, the way it is now is correct when the user needs both of these permissions